Jenkinsfile

```diff
@@ -171,6 +171,22 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
             }
         }
 
+        stage('QEMU Kola Run') {
+            utils.shwrap("""
+            coreos-assembler kola run || :
+            tar -cf - tmp/kola/ | xz -c9 > _kola_temp.tar.xz
+            """)
+            archiveArtifacts: "_kola_temp.tar.xz"
+        }
+
+        // archive the image if the tests failed
+        def report = readJSON file: "tmp/kola/reports/report.json"
+        if (report["result"] != "PASS") {
+            archiveArtifacts: "builds/${newBuildID}/fedora-coreos-${newBuildID}-qemu.qcow2"
```

Comment:
Let's use `builds/**/*.qcow2` instead to fish out the qemu image? There really should only be one in there, and it's less fragile than specifying the full filename, which could change. Actually, the path here for example is missing `${basearch}`.